.ddev/config.yaml

```diff
@@ -1,5 +1,5 @@
 name: lupus-decoupled
-type: drupal
+type: drupal11
```

Comment:
Does this need a ddev upgrade (which version)?

Reply:
reverted